include/nuttx/wireless/bluetooth/bt_uart.h

```diff
@@ -142,6 +150,10 @@ struct btuart_lowerhalf_s
 
   CODE int (*ioctl)(FAR const struct btuart_lowerhalf_s *lower,
                     int cmd, unsigned long arg);
+
+  /* Private instance data */
+
+  void *priv;
```

Comment:
Where `priv` is used?

Reply:
At least three places in the USB bluetooth driver.

1) When the USB bluetooth class driver is "constructed" this will be done:

  FAR struct usbhost_state_s *priv;
  /* Allocate a USB host class instance */
  priv = usbhost_allocclass();

 if (priv)
    {
          priv->lower.txacl     = usbhost_txacl;
          priv->lower.txcmd     = usbhost_txcmd;
          priv->lower.priv      = priv;  <----------------- store instance data to be used by txacl and txcmd

2) This is how txacl would use priv (with error checking removed for brevity)

static ssize_t usbhost_txacl(FAR const struct btuart_lowerhalf_s *lower,
                             FAR const void *buffer, size_t buflen)
{
  FAR struct usbhost_state_s *priv;
  FAR struct usbhost_hubport_s *hport;

  priv = (FAR struct usbhost_state_s*)lower->priv;  <--------------------
  hport = priv->usbclass.hport;
  nwritten = DRVR_TRANSFER(hport->drvr, priv->bulkout,
                               (uint8_t *)buffer, buflen);
   <--snip-->
  return 0;
}

Reply:
txcmd needs priv too.

Reply:
should we write usb bluetooth driver directly base on bt_driver_s?